src/Shared/Components/header/Breadcrumb.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { View, Text, Pressable } from "react-native";
+import { useNavigation, useRoute } from "@react-navigation/native";
+import MaterialIcons from "react-native-vector-icons/MaterialIcons";
+
+import styles from "../../styles";
+
+const Breadcrumb = ({ partLabel }) => {
```

Comment:
Needs comments on where props are coming from and/or what the mean

Reply:
Documentation added.
Changed to custom icon component.